v0/destinations/facebook_pixel/network-handler.js

```diff
@@ -59,14 +59,14 @@ const errorDetailsMap = {
         scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.API.SCOPE,
         meta: TRANSFORMER_METRIC.MEASUREMENT_TYPE.API.META.ABORTABLE
       }
-    },
-    33: {
-      status: 500,
-      statTags: {
-        scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.API.SCOPE,
-        meta: TRANSFORMER_METRIC.MEASUREMENT_TYPE.API.META.RETRYABLE
-      }
     }
+    // 33: {
+    //   status: 500,
+    //   statTags: {
+    //     scope: TRANSFORMER_METRIC.MEASUREMENT_TYPE.API.SCOPE,
+    //     meta: TRANSFORMER_METRIC.MEASUREMENT_TYPE.API.META.RETRYABLE
+    //   }
+    // }
```

Comment:
Can you add some comments regarding this commented code?

Reply:
@sanpj2292, how about we still keep this but just qualify it as abortable?

Reply:
@saikumarrs By default we're actually using `abortable` only. That is the reason I commented this. You can verify the same in `defaultStatTags` variable

Reply:
Since this is a valid and known status code, no harm in keeping it. Hence, my suggestion.

Reply:
Changed it